swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -59,224 +55,229 @@ CREATE TABLE Company.Employees (
+
+![Run the client](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/run-the-client.gif)
+
+If the program runs without throwing an error with the output below, that indicates that the connection has been established successfully. This client can be defined globally and used across all the parts of the program.
 
 ```
-$ bal run
-```
+Compiling source
+        featured_scenarios/build_a_data_service:0.1.0
 
-If the program runs without throwing an error, that indicates that the connection has been established successfully. This client can be defined globally and be used across all parts of the program.
+Running executable
+```
 
->**Info:** The MySQL package provides additional connection options and the ability to configure connection pool 
->properties when connecting to the database which, are not covered in this tutorial. To learn more about this, 
->see [`mysql:Client`](https://lib.ballerina.io/ballerinax/mysql/latest#Client).
+>**Info:** The MySQL package provides additional connection options and the ability to configure connection pool properties when connecting to the database, which are not covered in this tutorial. To learn more about this, see [`mysql:Client`](https://lib.ballerina.io/ballerinax/mysql/latest#Client).
 
 ### Execute the queries
 
 The `mysql:Client` provides two primary remote methods for performing queries.
 
 1. `query()` - Executes an SQL query and returns the results (rows) from the query. 
-   The `queryRow()` method is a variation of this method, which returns at most a single row from the result.
+   The `queryRow()` method is a variation of this method, which returns, at most, a single row from the result.
 
-2. `execute()` - Executes an SQL query and returns only the metadata of the execution.
+2. `execute()` - Executes an SQL query and returns only the execution metadata.
 
-To use the `query()`, `queryRow()`, and `execute()` methods, which can perform basic CRUD operations against the MySQL database, add the code below to the `main.bal` file. 
+#### Create the functions
 
-```ballerina
-isolated function addEmployee(Employee emp) returns int|error {
-    sql:ExecutionResult result = check dbClient->execute(`
-        INSERT INTO Employees (employee_id, first_name, last_name, email, phone,
-                               hire_date, manager_id, job_title)
-        VALUES (${emp.employee_id}, ${emp.first_name}, ${emp.last_name},  
-                ${emp.email}, ${emp.phone}, ${emp.hire_date}, ${emp.manager_id},
-                ${emp.job_title})
-    `);
-    int|string? lastInsertId = result.lastInsertId;
-    if lastInsertId is int {
-        return lastInsertId;
-    } else {
-        return error("Unable to obtain last insert ID");
-    }
-}
+Follow the steps below to create functions to use the `query()`, `queryRow()`, and `execute()` methods, which can perform basic CRUD operations against the MySQL database. 
```

Comment:
Instead of creating separate functions for the CRUD operations, we can directly create resource functions and include the DB operation? WDYT?

Reply:
Updated.